### DISCUSSION TITLE
blog/dev/2024-11-23-transaction-in-nodejs/

### DISCUSSION THREAD

**Discussion by giscus** [mingdodev/mingdodev.github.io#23] [Announcements]
# blog/dev/2024-11-23-transaction-in-nodejs/

moonshot

https://mingdodev.github.io/blog/dev/2024-11-23-transaction-in-nodejs/

**Comment by ywonchae1** [comment]
늦더라도 잊지 않고 회고하는 모습 너무 멋져용..🩷

**Reply by mingdodev** [reply]
마지막까지 같이 고민해줘서 고마웠고 너무너무 재밌었어요 🤍
